test/Microsoft.AspNetCore.Sockets.Client.Tests/LongPollingTransportTests.cs

```diff
@@ -201,5 +207,77 @@ public class LongPollingTransportTests
                 }
             }
         }
+
+        [Fact]
+        public async Task LongPollingTransportDispatchesMessagesReceivedFromPoll()
+        {
+            var message1Payload = new byte[] { (byte)'H', (byte)'e', (byte)'l', (byte)'l', (byte)'o' };
+            var message2Payload = new byte[] { (byte)'W', (byte)'o', (byte)'r', (byte)'l', (byte)'d' };
+            var encoded = Enumerable.SelectMany(new[] {
+                new byte[] { (byte)'B', 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x05, 0x00 }, message1Payload,
```

Comment:
I think there is a typo in the spec: https://github.com/aspnet/SignalR/pull/171/files#diff-42dec8df7724bd75eb93be084191c223R192 - the example shows `Text` as `0x80` and not `0x00`